src/components/atoms/CardContentRow/CardContentRow.tsx

```diff
@@ -0,0 +1,28 @@
+import * as styles from "./CardContentRow.css";
+
+import { Image } from "react-feather";
+
+export const CardContentRow: React.FC<{
+  title: string;
+  hasImage: boolean;
+  isNumbered: boolean;
+  num: number;
+  children: React.ReactNode;
+}> = ({ title, hasImage, isNumbered, num, children }) => {
+  return (
+    <div className={styles.row}>
+      {isNumbered ? <div className={styles.number}>{num}</div> : null}
+      <div className={styles.content}>
+        <div className={styles.title}>
+          <div className={styles.titleText}>{title}</div>
+          {hasImage ? (
+            <div className={styles.titleImage}>
+              <Image size={16} />
+            </div>
+          ) : null}
```

Comment:
여기도 아이콘을 div로 묶어줄 필요 없이 바로 style 적용하면 될 것 같습니당

```suggestion
          {hasImage && <Image size={16} className={styles.titleImage} />}
```

근데 아이콘 이름이 Image라 eslint에서 alt-text 넣어줘야 한다고 warning이 뜨네요.. 위에서 import 해올 때 alias 설정해주면 어떨까요?

```typescript
import { Image as ImageIcon } from "react-feather";
<ImageIcon />
```